blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,308 @@
+can focus on creating value to your users.
+
+But in reality, transferring aspects of your application to cloud services also means that now these
+aspects move from the category of "application" and become "infrastructure". In practice, it means
+that your code now treats these services as external entities, and managing them happens in separate
+tools, workflows, and oftentimes, in separate teams.
+
+So now, in order to add a route to my API gateway, I need to submit a ticket, or I have to update
+some YAML in some repo that I don't fully understand. And now, I can only test this change when my
+code reaches the shared staging environment? That's not good.
+
+So how can we realize the potential of the cloud, but also not be buried by its complexity? As a
+software developer, I'm sorry but I would have to say *abstractions*.
+
+One of my favorite historians, [Yuval Noah-Harari](https://en.wikipedia.org/wiki/Yuval_Noah_Harari)
+talks about how language’s ability to abstract and assign meaning enabled humans to establish common
+understandings and cooperative behaviors. Abstractions in software are no different.
+
+We believe that with the right abstractions, the cloud can fulfill its true potential. We believe it
+is possible to establish a better model for building and operating software on top of this powerful
+computing platform by establishing new common understandings and better collaboration within
+engineering organizations.
+
+### Introducing Wing Cloud
+
+We call this new abstraction [Wing Cloud](https://wing.cloud), and we are excited to share that
+we've partnered with an amazing group of investors such as [Battery
+Ventures](https://www.battery.com), [Grove Ventures](https://www.grovevc.com) and [StageOne
+Ventures](https://stageonevc.com/), as well as an incredible crew of [funds and
+angels](https://wing.cloud/about#backed-by) from across the industry on the journey to turn this
+idea into a reality.
+
+**So what is Wing Cloud?** It is a new kind of *abstract cloud*. It doesn’t involve data centers,
+machines, or provisioning engines. Instead, it’s a layer that enables builders to harness this
+general-purpose computing platform through a programming and operational model that unifies both
+infrastructure and application, and works across all cloud providers and services. It celebrates the
+choice and flexibility the cloud has to offer without compromising the developer experience and
+engineering practices.
+
+So, what does that look like in practice? Our vision for Wing Cloud consists of three main
+components:
+
+ * **Winglang** - A programming language for the cloud which unifies infrastructure and runtime code
+   under a single model.
+ * **Wing Cloud Library** - A high-level SDK for writing cloud applications that can be deployed to
+   all providers and run locally in a simulator
+ * **Wing Console** - A visual application-centric operations and management console.
```

Comment:
I still think it would be better to put the main blocks here so people who stop here get a fuller picture, and to stimulate others to continue to read the details.
What I miss here is a bullet point about the toolchain that helps developers with local development (testing, debugging, visualisation, etc.) and cutting iteration speed. It will also make more sense because you added subheadings below that talk about these in detail

```suggestion
  * ** Local Development Toolchain** - A set of tools for testing, debugging, visualizing and interacting with cloud applications locally in order to increase iteration speed.
  * **Wing Console** - A visual application-centric operations and management console.
```